README.md

```diff
@@ -62,6 +62,9 @@ kagglehub.model_download('google/bert/tensorFlow2/answer-equivalence-bem/1')
 
 # Download a single file.
 kagglehub.model_download('google/bert/tensorFlow2/answer-equivalence-bem', path='variables/variables.index')
+
+# Download a model or file, even if previously downloaded to cache. Only available outside Kaggle notebooks.
+kagglehub.model_download('google/bert/tensorFlow2/answer-equivalence-bem', force_download=True)
```

Comment:
Thanks for adding!

Eventually, I would like to generate & publish the generated python doc for this package.

And if we add more flags to the `model_download` function, then, we could simply link to the function full documentation for here.

No action now, just wanted to share some context with you.

Reply:
Sounds great, thanks!!